setup.md

```diff
@@ -14,27 +14,27 @@ The remaining tasks can be deferred until later in the development cycle, when y
 to test and iterate using a simulator or physical device.
 
 <table>
- <tr><th></th><th>Task</th><th>Required?</th></tr>
+ <tr><th>Task</th><th>Required?</th></tr>
  <tr>
-  <td align="center">&#10004;</td><td><a href="#system-requirements">Verify your system meets the requirements for Flutter and dependencies</a></td><td align="center">Y</td>
+  <td style="width:1600px;"><a href="#system-requirements">Verify your system meets the requirements for Flutter and dependencies</a></td><td align="center">Y</td>
  </tr>
  <tr>
-  <td align="center">&#10004;</td><td><a href="#get-the-flutter-sdk">Clone the Flutter repo</a></td><td align="center">Y</td>
+  <td><a href="#get-the-flutter-sdk">Clone the Flutter repo</a></td><td align="center">Y</td>
  </tr>
  <tr>
-  <td align="center">&#10004;</td><td><a href="#get-the-flutter-sdk">Update your path</a></td><td align="center">Y</td>
+  <td><a href="#get-the-flutter-sdk">Update your path</a></td><td align="center">Y</td>
```

Comment:
As intended. This is a safe way to override a site-wide CSS style that makes the first td of every table 150px wide - very narrow indeed. It's needed to make this table look normal for First Look. 

After First Look, we'll diagnose and fix main.css.